ParalayoutTests/InterpolationTests.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Copyright © 2017 Square, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Paralayout
+import XCTest
+
+
+class InterpolationTests: XCTestCase {
+
+    func testCGRectInterpolation() {
+        let startRect = CGRect(x: 0, y: 0, width: 5, height: 5)
+        let endRect = CGRect(x: 0, y: 0, width: 10, height: 10)
```

Comment:
Let's use a sample where the `origin` is non-zero and the midpoint of the widths and height aren't the same. That will give us better validation that all of the members are being interpolated correctly.

Reply:
I often like to prefix the numbers differently. For example, `(1000, 2000, 3000, 4000) -> (1050, 2050, 3050, 4050)` makes it easy to verify that the result isn't mixing up any of the values.